src/floating-focus.js

```diff
@@ -226,4 +235,9 @@ export default class FloatingFocus {
 	repositionElement(target, floater) {
 		Object.assign(floater.style, this.getFloaterPosition(target));
 	}
+
+	remove(){
```

Comment:
We activate the floating focus effect by instantiating this with its constructor, so therefore I find it a bit strange to have a `remove()` method like this which then leaves the class sort of floating in space.
How about renaming this to `deactivate()` or `disable()` or something similar, and then also implementing an `activate()` or `enable()` method which can enable it again? (In the end that would only call `addEventListeners` again but it makes it clearer from the outside what you're doing)

Reply:
I think you are right, it is clearer and more personalized.